app/serializers/garden_serializer.rb

```diff
@@ -1,9 +1,11 @@
 class GardenSerializer
   include JSONAPI::Serializer
-  attributes :name, :zip_code, :state_code, :weather_forecast
+  attributes :name, :zip_code, :state_code, :user_id
   has_many :plots
 
-  attribute :weather_forecast do |object|
-    WeatherFacade.get_forecast(object.zip_code)
+  attribute :weather_forecast do |object, params|
+    if params[:weather].present? && params[:weather]
```

Comment:
I'm wondering why both of these are included, aren't they essentially asking the same thing?